cytotable/presets.py

```diff
@@ -203,11 +195,11 @@
                 cytoplasm.Metadata_TableNumber = image.Metadata_TableNumber
                 AND cytoplasm.Metadata_ImageNumber = image.Metadata_ImageNumber
             LEFT JOIN read_parquet('cells.parquet') AS cells ON
-                cells.Metadata_TableNumber = cytoplasm.Metadata_TableNumber
+                cells.Metadata_TableNumber = cells.Metadata_TableNumber
```

Comment:
This seems off. Wanting to confirm this is an intended change.

Reply:
Thanks, yes this is incorrect, I'll update these references.